config/widgets/settings/battleOptionsTab.json

```diff
@@ -28,7 +28,7 @@
 				},
 				{
 					"text": "core.genrltxt.397", // Creature info
-					"position": {"x": 10, "y": 205}
+					"position": {"x": 10, "y": 240}
```

Comment:
Other values changes by 30px, but this one by 35px. Intended?

Reply:
Probably not. Nice catch!
Corrected.